.github/workflows/bvt-appleclang.yml

```diff
@@ -7,23 +7,29 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
+    - name: install meson
+      run: pipx install meson
+
     - name: check compiler versions
       run: |
-        clang --version
+        cc --version
         xcodebuild -version
 
-    - name: build and run test with AppleClang
+    - name: build and run test with AppleClang on cmake
       run: |
         cmake -B build -GNinja -DCMAKE_CXX_STANDARD=23 -DCMAKE_BUILD_TYPE=Release -DPROXY_BUILD_MODULES=FALSE
         cmake --build build -j
         ctest --test-dir build -j
         mkdir build/drop
-        chmod +x tools/dump_build_env.sh
-        ./tools/dump_build_env.sh clang++ build/drop/env-info.json
+        bash ./tools/dump_build_env.sh c++ build/drop/env-info.json
 
-    - name: run benchmarks
+    - name: build and run test with AppleClang on meson
       run: |
-        build/benchmarks/msft_proxy_benchmarks --benchmark_min_warmup_time=0.1 --benchmark_min_time=0.1s --benchmark_repetitions=30 --benchmark_enable_random_interleaving=true --benchmark_report_aggregates_only=true --benchmark_format=json > build/drop/benchmarking-results.json
+        meson setup builddir --buildtype=release -Dtests=enabled
```

Comment:
From the CI log, it seems that `benchmarks` is not built. While benchmarks only need to execute once, it should have minimal build coverage for Meson.